flocker/node/script.py

```diff
@@ -301,13 +223,10 @@ class ServeScript(object):
     a Flocker cluster.
     """
     def main(self, reactor, options, volume_service):
-        api_service = create_api_service(
-            TCP4ServerEndpoint(reactor, options["port"]))
-        parent_service = _ServeService(volume_service, api_service)
-        return _main_for_service(reactor, parent_service)
+        return main_for_service(reactor, volume_service)
 
 
-def flocker_serve_main():
+def flocker_volume_main():
```

Comment:
We've been talking about "agents".  Perhaps this should be something like `flocker_volume_agent_main`?  Or maybe `flocker_volume_convergence_main`.  Although since it doesn't do any converging yet I'm not quite sure if that's the right level of abstraction.